mixins/form.jsx

```diff
@@ -272,7 +309,15 @@ module.exports = {
     var self = this;
     var props = {};
     fields.forEach(function(name) {
-      props = assign(props, self.state[name].state.values);
+      var prop = self.state[name].state.values;
+      // Currently some fields expose their values on the form, and not themselves.
+      // So we need to check for props in both places until all field componenets are updated.
+      if (!prop) {
+        prop = self.state.props[name].values;
+      }
+      if (prop) {
```

Comment:
do you really need this check now? it looks like you are going to have prop no matter what?